jailbreaking/post-jailbreak/setting-up-a-hotfix.md

```diff
@@ -22,7 +22,19 @@ A hotfix allows your Kindle's jailbreak to persist after updating. There are two
 
 {: .note }
 > You will need to run the hotfix after every update
+>
 
+# Troubleshooting
+
+## Impossible update hotfix
+There is a good chance that after installing an older version of HotFix you immediately disabled OTA updates, so the update file may be ignored!
```

Comment:
HotFix -> hotfix
I would also mention that you can install the hotfix using MRPI